lib/examples/compare-chromedp/upload/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/ysmood/rod"
+	"github.com/ysmood/rod/lib/launcher"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+)
+
+var flagPort = flag.Int("port", 8544, "port")
+
+// This example demonstrates how to upload a file on a form.
+func main() {
+	flag.Parse()
+
+	// get wd
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	filepath := wd + "/main.go"
+
+	// get some info about the file
+	fi, err := os.Stat(filepath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// start upload server
+	result := make(chan int, 1)
+	go uploadServer(fmt.Sprintf(":%d", *flagPort), result)
+
+	url := launcher.New().Headless(false).Launch()
+	browser := rod.New().ControlURL(url).Connect()
+
+	page := browser.Page(fmt.Sprintf("http://localhost:%d", *flagPort))
+
+	page.Element(`input[name="upload"]`).Input(filepath)
```

Comment:
It's better to use `SetFiles(filepath)` here. So that you don't have to use `os.Getwd()` at all.

Reply:
Sure, I can do that for uploading, but how about when logging the file on line 48?